pkg/webui/components/form/field/tooltip.js

```diff
@@ -0,0 +1,97 @@
+// Copyright © 2021 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { defineMessages } from 'react-intl'
+
+import Tooltip from '@ttn-lw/components/tooltip'
+import Icon from '@ttn-lw/components/icon'
+import Link from '@ttn-lw/components/link'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import PropTypes from '@ttn-lw/lib/prop-types'
+import fieldDescriptions from '@ttn-lw/lib/field-description-messages'
+
+import style from './field.styl'
+
+const m = defineMessages({
+  descriptionTitle: 'What is it?',
+  locationTitle: 'Where can I find it?',
+  absenceTitle: "What if I can't find it?",
```

Comment:
```suggestion
  descriptionTitle: 'What is this?',
  locationTitle: 'What should I enter here?',
  absenceTitle: "What if I cannot find the correct value?",
```

Sorry, this deviates from the wireframe but I realized this needs to be formulated a bit broader.